### PR TITLE
test sonar

### PR DESCRIPTION
**Pull request by dieppa**

### CODE REVIEW

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java [hunk 1]
```diff
@@ -110,6 +112,19 @@ void execute() throws MongockException {
 
   }
 
+  //this is to force sonar issue
+  Object ex;
+  private Object getException() {
+    if(ex == null) {
+      synchronized (this) {
```
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: BLOCKER Remove this dangerous instance of double-checked locking. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java [hunk 2]
```diff
@@ -91,6 +91,8 @@ void execute() throws MongockException {
       return;
     }
 
+    Object o = getException();
```
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: MAJOR Remove this useless assignment to local variable "o". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "o" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

**Comment by dieppa** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L150](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L150): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L191](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L191): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java [hunk 3]
```diff
@@ -110,6 +112,19 @@ void execute() throws MongockException {
 
   }
 
+  //this is to force sonar issue
+  Object ex;
```
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by dieppa** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L150](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L150): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L191](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L191): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java (hunk 1 above)
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: BLOCKER Remove this dangerous instance of double-checked locking. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)

**Comment by dieppa** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L150](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L150): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L191](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L191): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by dieppa** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L135](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L135): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L176](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L176): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by cloudyrock-tech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L135](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L135): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L176](https://github.com/cloudyrock/mongock/blob/81b363805d718099c6468b9e41c5749ff82c21c9/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L176): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java [hunk 4]
```diff
@@ -107,7 +109,19 @@ void execute() throws MongockException {
       lockChecker.releaseLockDefault();//we do it anyway, it's idempotent
       logger.info("Mongock has finished his job.");
     }
+  }
 
+  //this is to force sonar issue
+  Object ex;
+  private Object getException() {
+    if(ex == null) {
+      synchronized (this) {
```
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: BLOCKER Remove this dangerous instance of double-checked locking. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java (hunk 2 above)
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: MAJOR Remove this useless assignment to local variable "o". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "o" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

**Comment by dieppa** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/a12b70afb031299f6064822b88023dfb2217f2d4/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L149](https://github.com/cloudyrock/mongock/blob/a12b70afb031299f6064822b88023dfb2217f2d4/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L149): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L190](https://github.com/cloudyrock/mongock/blob/a12b70afb031299f6064822b88023dfb2217f2d4/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L190): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by dieppa** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/0513c9eaebfe3c906c58adfd918fa7a32af951d2/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L149](https://github.com/cloudyrock/mongock/blob/0513c9eaebfe3c906c58adfd918fa7a32af951d2/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L149): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L190](https://github.com/cloudyrock/mongock/blob/0513c9eaebfe3c906c58adfd918fa7a32af951d2/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L190): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by dieppa** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/ddd024c8a139422566fd82a0d0d2c3011888dde1/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L134](https://github.com/cloudyrock/mongock/blob/ddd024c8a139422566fd82a0d0d2c3011888dde1/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L134): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L175](https://github.com/cloudyrock/mongock/blob/ddd024c8a139422566fd82a0d0d2c3011888dde1/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L175): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java (hunk 4 above)
Comment (cloudyrock-tech) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java:118]: BLOCKER Remove this dangerous instance of double-checked locking. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java (hunk 2 above)
Comment (cloudyrock-tech) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java:94]: MAJOR Remove this useless assignment to local variable "o". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "o" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

**Comment by cloudyrock-tech** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/ddd024c8a139422566fd82a0d0d2c3011888dde1/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L149](https://github.com/cloudyrock/mongock/blob/ddd024c8a139422566fd82a0d0d2c3011888dde1/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L149): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L190](https://github.com/cloudyrock/mongock/blob/ddd024c8a139422566fd82a0d0d2c3011888dde1/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L190): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by cloudyrock-tech** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L150](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L150): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L191](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L191): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java [hunk 5]
```diff
@@ -107,9 +109,22 @@ void execute() throws MongockException {
...
```
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: BLOCKER Remove this dangerous instance of double-checked locking. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)

File: src/main/java/com/github/cloudyrock/mongock/Mongock.java (hunk 2 above)
Comment (dieppa) [inline src/main/java/com/github/cloudyrock/mongock/Mongock.java]: MAJOR Remove this useless assignment to local variable "o". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "o" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

**Comment by dieppa** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L150](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L150): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L191](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L191): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by dieppa** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L136](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L136): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L177](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L177): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by cloudyrock-tech** [comment]
SonarQube analysis reported 6 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [Mongock.java#L88](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L88): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L150](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L150): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [Mongock.java#L191](https://github.com/cloudyrock/mongock/blob/4b6bab9cbc783fc0295df69b71770ec4abd4cd50/src/main/java/com/github/cloudyrock/mongock/Mongock.java#L191): Remove the declaration of thrown exception 'com.github.cloudyrock.mongock.MongockException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request closed without merging**